docs/_documentation/fundamentals/data-binding.md

```diff
@@ -76,22 +74,16 @@ For example:
 private bool _rememberMe;
 public bool RememberMe
 {
-    get {
-        return _rememberMe;
-    }
-    set
-    {
-        _rememberMe = value;
-        RaisePropertyChanged(() => RememberMe);
-    }
+    get => _rememberMe;
+    set => SetProperty(ref _rememberMe, value);
 }
 ```
 
 - then a binding might connect together `IsChecked` on the View with `RememberMe` in the ViewModel.
 
 ### DataBound events and actions
 
-Databinding also enables a ViewModel to react to 'events' which occur in a View - e.g. for a ViewModel to respond to events such as a button being pressed.
+Data binding also enables a ViewModel to react to 'events' which occur in a View - e.g. for a ViewModel to respond to events such as a button being pressed.
```

Comment:
Data binding, data-binding or databinding? [Google] uses "Data binding" it seems (https://developer.android.com/topic/libraries/data-binding/index.html)

Reply:
But in other parts in this document we uses "Databinding"

Reply:
Right, will consolidate this ;)